apps/ideas/forms.py

```diff
@@ -177,6 +182,25 @@ class Meta:
             'members'
         ]
 
+    def __init__(self, *args, **kwargs):
+        self.request = kwargs.pop('request', None)
+        super().__init__(*args, **kwargs)
+        help_pages = HelpPages.for_site(self.request.site)
+        if (help_pages.annual_theme_help_page and
+                help_pages.annual_theme_help_page.live):
+            url = help_pages.annual_theme_help_page.url
+            current_outcome_help_text = self.fields['outcome'].help_text
```

Comment:
Maybe creating the link and bundling it up to a new help text, could even be helper function.